cypress/integration/SEPs.ts

```diff
@@ -12,7 +12,7 @@ function readWriteReview() {
 
   cy.finishedLoading();
 
-  cy.get('@dialog').contains('Proposal information');
+  cy.get('@dialog').contains('Proposal Information');
```

Comment:
You could do `cy.get('@dialog').contains('Proposal information', { matchCase: false });` so it's easier to maintain, unless you want to test the case sensitivity

Reply:
Yes maybe that is even better option. I just found that test failing randomly and I think it was because of not matching the case.